tasks.py

```diff
@@ -1,16 +1,142 @@
-from colored import attr
-from colored import fg
-from colored import stylize
+import os
+from contextlib import contextmanager
 from invoke import task
+import invoke
 
-ECHO_STYLE = fg("blue") + attr("bold")
-DB_NAME = "fsd_assess_store_test"
+@contextmanager
+def env_var(key, value):
+    old_val = os.environ.get(key, "")
+    os.environ[key] = value
+    yield
+    os.environ[key] = old_val
 
+def echo_print(to_print):
+
+    from colored import attr
+    from colored import fg
+    from colored import stylize
+
+    ECHO_STYLE = fg("blue") + attr("bold")
+
+    print(stylize(to_print, ECHO_STYLE))
+
+def error_print(to_print):
+
+    from colored import attr
+    from colored import fg
+    from colored import stylize
+
+    ECHO_STYLE = fg("red") + attr("bold")
+
+    print(stylize(to_print, ECHO_STYLE))
+
+def echo_input(to_print):
+
+    from colored import attr
+    from colored import fg
+    from colored import stylize
+
+    ECHO_STYLE = fg("blue") + attr("bold")
+
+    return input(stylize(to_print, ECHO_STYLE))
 
 @task
-def bootstrap_test_db(c, database_host="localhost"):
-    """Create a clean database for testing"""
-    c.run(f"dropdb -h {database_host} --if-exists {DB_NAME}")
-    print(stylize(f"{DB_NAME} db dropped...", ECHO_STYLE))
-    c.run(f"createdb -h {database_host} {DB_NAME}")
-    print(stylize(f"{DB_NAME} db created...", ECHO_STYLE))
+def profile_pytest(c, testrows=100, resultsfile="profile.txt"):
+    """Runs profiling during pytest to help find slow code."""
+
+    if not resultsfile.endswith(".txt"):
+        resultsfile = resultsfile.split(".")
+        resultsfile = resultsfile[0] + ".txt"
+    echo_print("Profiling tests.")
+    c.run(
+        "python -m cProfile -s tottime -m pytest"
+        f" > {resultsfile}"
+    )
+    echo_print(f"Results saved in f{resultsfile}")
+
+
+@task
+def bootstrap_dev_db(c):
```

Comment:
Are these tasks documented somewhere? When are they intended to be used? Do they need adding to the readme?

Reply:
Writing up READMEs now